tests/linalg_test.py

```diff
@@ -559,6 +559,27 @@ def args_maker():
                             check_dtypes=True, tol=1e-3)
     self._CompileAndCheck(np.linalg.inv, args_maker, check_dtypes=True)
 
+    @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name":
+       "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+       "shape": shape, "dtype": dtype, "rng": rng}
+      for shape in [(1, 1), (4, 4), (2, 5, 5), (200, 200), (5, 5, 5)]
+      for dtype in float_types
```

Comment:
Please add tests covering:
- Complex dtypes. I see at least one place where your code is missing `.conjugate()` vs NumPy's implementation, so I guess this would currently fail.
- More array shapes, at least including a wide matrix and a tall matrix.